src/main/java/nl/tudelft/contextproject/Main.java

```diff
@@ -118,11 +119,16 @@ protected void setInputManager(InputManager im) {
 		inputManager = im;
 	}
 
+
 	@Override
 	public void simpleInitApp() {
+		tickListeners = new LinkedList<>();
 		setDisplayFps(debugHud);
 		setDisplayStatView(debugHud);
-		getFlyByCamera().setMoveSpeed(50);
+		
+		flyCam.setMoveSpeed(100);
+		viewPort.setBackgroundColor(new ColorRGBA(0.1f, 0.1f, 0.1f, 1f));

```

Comment:
Why not use getFlyByCamera? The getters can be properly mocked, the field cannot.
In addition, jMonkeyVR does some strange stuff with cameras, and explicitly mentions that you should use the getters for proper VR support.